src/styles/colour/index.md.njk

```diff
@@ -14,16 +14,21 @@ You must make sure that the contrast ratio of text and interactive elements in
 your service meets [level AA of the Web Content Accessibility Guidelines
 (WCAG 2.0)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
 
-## Sass variables
-Use the Sass variables listed below, so you’ll automatically get any updates to
-the GOV.UK colour palette when you update GOV.UK Frontend.
-
 ## Main colours
 
+Use the Sass variables provided rather than copying the hexadecimal colour
+codes. This means that your service will always use the most recent colour
+palette when you update GOV.UK Frontend.
+
+Avoid using these variables to re-use colours in different contexts. For
```

Comment:
```suggestion
Avoid using these variables to apply the colours in different contexts from the ones they are intended for. For
```

Reply:
If you don't accept this change and keep the former wording, can you change 're-use' to 'reuse'?